desktop-app/index.html

```diff
@@ -237,6 +236,18 @@ <h3 class="fader" style="display: inline;">{{user.distance_mi}} miles away, Acti
     <h1>Settings</h1>
     <form name="tppSettings">
       <fieldset>
+        <legend>Location</legend>
+			  <p>
+				<a href="" ng-click="toggleLocation()" >{{getCookie('currentCity') ? getCookie('currentCity') : 'set location'}}</a>
```

Comment:
PR looks good! Would you mind capitalizing to 'Set Location' to match the other settings?